packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox-group/gux-form-field-checkbox-group.tsx

```diff
@@ -94,6 +94,13 @@ export class GuxFormFieldCheckboxGroupBeta {
     if (checkboxSlots) {
       checkboxSlots.forEach(item => {
         item.hasGroupError = hasGroupError;
+
+        const checkboxInput = item.querySelector('input');
+        if (hasGroupError) {
+          checkboxInput.setAttribute('aria-invalid', 'true');
+        } else {
+          checkboxInput.removeAttribute('aria-invalid');
+        }
```

Comment:
Would it be better to handle this in the `gux-form-field-checkbox` component.
ie track the `hasGroupError` property in that component.

Reply:
Now handling in the `checkbox` component instead.